### PR TITLE
Private/gokay/notes previews

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: wsd/ClientSession.hpp [hunk 1]
```diff
@@ -50,12 +50,6 @@ class ClientSession final : public Session
                WAIT_DISCONNECT // closed and waiting for Kit's disconnected message
     );
 
-    enum ViewMode {
```
Comment (eszkadev) [inline wsd/ClientSession.hpp:53]: this is useful, can we reintroduce that enum? :)

File: browser/src/layer/tile/ImpressTileLayer.js [hunk 2]
```diff
@@ -61,6 +61,12 @@ L.ImpressTileLayer = L.CanvasTileLayer.extend({
 			else
 				targetElement.classList.remove('selected');
```
Comment (eszkadev) [inline browser/src/layer/tile/ImpressTileLayer.js:62]: Unrelated but I noticed: an ugly state handler - we could do it automatically with state update for an uno command. Do we send notes command state from core? we should.

**Review by eszkadev — APPROVED** [review]

File: browser/src/layer/tile/ImpressTileLayer.js (hunk 2 above)
Comment (gokaysatir) [reply browser/src/layer/tile/ImpressTileLayer.js:62]: Yeah but i think that's the way we use for toggle buttons. But still, we may consider a better approach as you say. I think it'd take some time so i am passing it for now :)

**Comment by gokaysatir** [comment]
Thanks for checking :)

File: wsd/ClientSession.hpp (hunk 1 above)
Comment (gokaysatir) [reply wsd/ClientSession.hpp:53]: Ah, yes, let me note this also. I will check these along with the JSON API.

**Pull request merged**